docs/Project.toml

```
@@ -0,0 +1,5 @@
+[deps]
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+
+[compat]
+Documenter = "~0.20"
```

```suggestion
Documenter = "~0.22"
```
Might as well go newest